src/reference/modules/installation/pages/operating_systems.adoc

```diff
@@ -18,7 +18,7 @@ available freely for these platforms:
 | OS | Version | Architecture
 
 | Debian | 10 and 11 | 64bit
-| Ubuntu | 18.04 LTS and 20.04 LTS | 64bit
```

Comment:
you need to keep ubuntu 20.04

Reply:
18.04 to 22.04 is ok ?

Reply:
no it may induced that we suport non LTS version